source/MapPanel.h

```diff
@@ -42,6 +42,16 @@ class System;
 // the systems based on a selected criterion. It also handles finding and
 // drawing routes in between systems.
 class MapPanel : public Panel {
+private:
+	struct WormholeArrow {
```

Comment:
This doesn't need to be here, move it to the anonymous namespace in the cpp file.